azure-toolkit-libs/azure-toolkit-common-lib/src/main/java/com/microsoft/azure/toolkit/lib/common/messager/IAzureMessager.java

```diff
@@ -0,0 +1,32 @@
+package com.microsoft.azure.toolkit.lib.common.messager;
+
+import javax.annotation.Nonnull;
+
+public interface IAzureMessager {
+
+    default boolean confirm(@Nonnull String message, String... title) {
+        return false;
+    }
+
+    default void alert(@Nonnull String message, String... title) {
+    }
+
+    default void success(@Nonnull String message, String... title) {
+    }
+
+    default void info(@Nonnull String message, String... title) {
+    }
+
+    default void warning(@Nonnull String message, String... title) {
+    }
+
+    default void error(@Nonnull String message, String... title) {
+    }
+
+    default void error(@Nonnull Throwable throwable, String... title) {
+    }
+
+    default String value(String val) {
```

Comment:
what's this for?

Reply:
to wrap value so that value will be highlighted, e.g. in Maven, it would be `TextUtils.cyan(val)`